doc/changelog/05-deprecated/1398-semi-matrix.md

```diff
@@ -0,0 +1,5 @@
+- in `matrix.v`
+  + lemmas `const_mx_is_zmod_morphism`, `swizzle_mx_is_zmod_morphism`,
+    `diag_mx_is_zmod_morphism`, `scalar_mx_is_zmod_morphism`,
+    `mxtrace_is_zmod_morphism` (use `raddfB` instead)
+    ([#1398](https://github.com/math-comp/math-comp/pull/1398)).
```

Comment:
It looks a bit weird that we are going to introduce new constants and then deprecate them in the same release (2.5). But, if we remove them, we probably need to amend `doc/changelog/03-renamed/1296-1first.md`.